### DISCUSSION TITLE
[Question]: 0.14.0 Beta 1 - Filter/sort/review on zone activity

### DISCUSSION THREAD

**Discussion by tubalainen** [blakeblackshear/frigate#11664] [Ask A Question] [answered]
### What is your question:

I cannot find any options/filters etc to filter on zones.

According to the release notes this should be possible, I cannot find any config options that ive missed nor any button in the new UI that I havnt tried. 

Thanks :)

**Comment by NickM-27** [comment, marked as answer]
you use zones to control what qualifies as an alert vs detection. Currently there are no zone filter options in the UI

**Reply by tubalainen** [reply]
> you use zones to control what qualifies as an alert vs detection. Currently there are no zone filter options in the UI

Okay thanks for the info.
Shall I make a official freature req or is this already on your radar for the upcoming releases? :)

Thanks a billion.

**Reply by pseudoacacia** [reply]
While the alert/detection is helpful for casual review (when I can generally ignore "detections"), it's not nearly granular enough for me. Recent example: A yard decoration was missing and I wasn't sure how long it had been gone. Thankfully, I have a fairly small zone that includes it, so by filtering to that zone I could very quickly determine when it disappeared. With the new UI, I'd have to look through piles of alerts that were triggered elsewhere in frame. 

I hope the ability to filter/search based on zones is coming back soon. It's the only thing I've really missed since trying out the 0.14.0 Beta 1.

**Reply by NickM-27** [reply]
for this use case I would suggest it would be better to not use alerts / detections at all. You'd most likely have much quicker results by just using the motion page and scrubbing or clicking on the timeline to find when the decoration went missing. This has happened to me a couple times and I found the answer very quickly this way

**Reply by hawkeye217** [reply]
Similar thing happened to me with my trash bin in a rainstorm the other day. 

The more intuitive way of finding when this happened is to simply scrub a timeline around the time of the rainstorm. This was impossible in 0.13, so we just got used to what we had available - filtering events by camera or by zone, and clicking through a bunch of short clips, hoping we'd see the right one.

After we developed the scrubbable timeline for motion review and we started using it, it only took a short time for my brain's switch to flip. It really does make things so much easier.

**Reply by pseudoacacia** [reply]
Can't scrub around the time it goes missing without any idea of when it went missing (in this case, it turns out it had been gone nearly a week, and the camera was installed and the zone created because things have been stolen in the past, but there are other zones of equally alert-level attention in frame). I've done that sort of searching before, too, and while a binary-search approach eventually narrows it down, it's frustrating to have to take such an approach when I know that the information exists to pin point what I'm looking for but I just can't access it through the UI.

Other situations that I've used the zone filters for include noticing damage on a retaining wall by the driveway and looking to find out whose car hit it. Since my driveway is a loop with two entrances (and both visible from the same camera), filtering to the zone that covers the relevant entrance narrowed the search immensely (that entrance is less often used, so it was the third most recent car in that zone but 53rd most recent car entering either branch of the driveway so filtering by "car" would be helpful in removing the hundreds of "person" and other detections, but not nearly as helpful as "car"+"west entrance" was).

So, sure, if I have a reasonable window of time to scrub through, that might work better (but glancing through a bunch of screenshots focused on activity in the specific region is much fast than reviewing live video, both in terms of loading time but also in ignoring irrelevant times...if the scrubbing timeline could be filtered to highlight activity in specific zone that might work out as superior).

Anyway, I really hope we will regain the ability to filter by zone.

**Reply by Mincka** [reply]
I also asked for zone filtering. It was a useful feature in 0.13.

In my case, I want to check back alerts and detections few days after they were reviewed, but since there's a lot of them, I want to filter them for a specific object type and zone only. Alert vs Detect does not help to separate them with only the zone of interest in one of them.

**Reply by tubalainen** [reply]
I used zone filters alot in 0.13. I would love to see it back in the new UI.

**Comment by andrey-skat** [comment]
Filtering by zone is one of the important features in 0.13. It's more useful than filtering by label and even by camera.
I rarely need to review all events, usually, it's needed in some important zones. 

And I have zones with the same name that exist in different cameras.
So, I made a quick filter just for them: gates, entrance door (among others). Sometimes I need the opposite - to exclude the noisy street zone from review.

Another example. A camera covers the street, the area near the entrance, and parking. I want to filter only the area near the entrance (who walked close to the entrance or stood there). Without a filter, it can't be done. Manually scrubbing through many hours of timeline is not an option :)

**Comment by tubalainen** [comment]
Created a Feature Request, [here](https://github.com/blakeblackshear/frigate/issues/11796)
